src/coreclr/tools/Common/JitInterface/ThunkGenerator/ThunkGenerator.csproj

```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
+
+    <!-- TODO: run ThunkGenerator on CI to detect new code analysis issues early -->
+    <RunAnalyzers>false</RunAnalyzers>
```

Comment:
revert?

Reply:
It's an intentional change - ThunkGenerator is not built/run on CI so we brake it time to time when enable more diagnostics

Reply:
Ok, so it is to solve this purpose? https://github.com/dotnet/runtime/pull/82973#discussion_r1159191393

Reply:
> Ok, so it is to solve this purpose? [#82973 (comment)](https://github.com/dotnet/runtime/pull/82973#discussion_r1159191393)

No, in this case `gen.bat` sometimes fails to build and complains on new analyzers. Current issues have just been fixed in https://github.com/dotnet/runtime/pull/84390 but they can appear again in future